core/primitives/src/block_header.rs

```diff
@@ -493,6 +591,7 @@ impl BlockHeader {
         block_merkle_root: CryptoHash,
         prev_height: BlockHeight,
         clock: near_time::Clock,
+        chunk_endorsements: Option<ChunkEndorsementsBitmap>,
```

Comment:
This function and its `genesis` version became really big... Could you leave one or two latest versions of block headers on top and hide old ones into something like `old_impl`?

Reply:
done.